guides/common/modules/proc_creating-a-host.adoc

```diff
@@ -1,102 +1,4 @@
-If you want to use non Red Hat operating systems, select *All Media*, then select the installation media from the *Media Selection* list.
-You can select a partition table from the list or enter a custom partition table in the *Custom partition table* field.
-You cannot specify both.
-. On the *Parameters* tab, click *Add Parameter* to add any parameter variables that you want to pass to job templates at run time.
-This includes all Puppet Class, Ansible Playbook parameters and host parameters that you want to associate with the host.
-To use a parameter variable with an Ansible job template, you must add a *Host Parameter*.
-ifdef::satellite[]
-+
-When you create a host, you can set system purpose attributes.
-System purpose attributes help determine which repositories are available on the host.
-System purpose attributes also help with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
-In the *Host Parameters* area, enter the following parameter names with the corresponding values.
-ifndef::orcharhino[]
-For the list of values, see {RHELDocsBaseURL}8/html/automatically_installing_rhel/proc_configuring-system-purpose-using-the-subscription-manager-command-line-tool_rhel-installer[Configuring System Purpose using the subscription-manager command-line tool] in _Automatically installing RHEL{nbsp}8_.
-endif::[]
-+
-* `syspurpose_role`
-* `syspurpose_sla`
-* `syspurpose_usage`
-* `syspurpose_addons`
-endif::[]
-
-+
-If you want to create a host with pull mode for remote job execution, add the `enable-remote-execution-pull` parameter with type `boolean` set to `true`.
-For more information, see xref:transport-modes-for-remote-execution_{context}[].
-. On the *Additional Information* tab, enter additional information about the host.
-. Click *Submit* to complete your provisioning request.
-
-[id="cli-creating-a-host_{context}"]
-.CLI procedure
-
-* To create a host associated to a host group, enter the following command:
-+
-[options="nowrap", subs="+quotes,attributes"]
-----
-# hammer host create \
---ask-root-password yes \
---hostgroup "_My_Host_Group_" \
---interface="primary=true, \
-            provision=true, \
-            mac=_My_MAC_Address_, \
-            ip=_My_IP_Address_" \
---location "_My_Location_" \
---name "_My_Host_Name_" \
---organization "_My_Organization_"
-----
-+
-This command prompts you to specify the root password.
-It is required to specify the host's IP and MAC address.
-Other properties of the primary network interface can be inherited from the host group or set using the `--subnet`, and `--domain` parameters.
-You can set additional interfaces using the `--interface` option, which accepts a list of key-value pairs.
-For the list of available interface settings, enter the `hammer host create --help` command.
+Create a host using the procedure specific to your compute resource in the {ProvisioningDocURL}[{ProvisioningDocTitle}].
```

Comment:
Creating a host is part of the host provisioning process.
There are multiple provisioning methods and each requires different configuration of {Project} and {SmartProxies}.
For more information, see {ProvisioningDocURL}[{ProvisioningDocTitle}].

Reply:
```suggestion
Creating a host is part of the host provisioning process.
There are multiple provisioning methods and each requires different configuration of {Project} and {SmartProxies}.
For more information, see {ProvisioningDocURL}[{ProvisioningDocTitle}].
```